app/models/application_authentication.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationAuthentication < ApplicationRecord
```

Comment:
I think we need a `before_validation` hook to set the tenant also.

Reply:
A request spec covering `POST /api/v2.0/application_authentication` should cover that

Reply:
:+1:

Reply:
Added https://github.com/RedHatInsights/sources-api/pull/184/files#diff-aca3e3260b70e2940cde506fd8bf2f7fR6 and added https://github.com/RedHatInsights/sources-api/pull/184/files#diff-2b415817a335813d2379e808df1b8593R49